src/Sulu/Bundle/DocumentManagerBundle/Bridge/DocumentInspector.php

```diff
@@ -81,7 +82,7 @@ public function getWebspace($document)
      *
      * @return string
      */
-    public function getContentPath(StructureBehavior $document)
+    public function getContentPath(PathBehavior $document)
     {
         $path = $this->getPath($document);
```

Comment:
hmm the path is working for all documents ... maybe we should remove the typehint at all.